### ISSUE TITLE
Confusion over private property location

### ISSUE THREAD

**Issue by muglug** [vimeo/psalm#440]
```php
class A {
  /** @var string */
  private $foo;
  
  public function __construct(string $foo) {
    $this->foo = $foo;
  }
}

class B extends A {
  /** @var string */
  private $foo;
  
  public function __construct(string $foo) {
    $this->foo = $foo;
    parent::__construct($foo);
  }
}
```

Expected: no error
Actual: `InaccessibleProperty` at the first `$this->foo = $foo;` in `A`

**Comment by muglug** [comment]
Fixed in f3bfb089ad467a303d4fafad1b342da2bdac9d7a
